tests/integration/test_db_router.py

```diff
@@ -239,9 +196,41 @@ async def test_keystone_bundle_db_router(ops_test: OpsTest) -> None:
         )
 
         # Deploy and test another deployment of keystone
-        await deploy_and_relate_keystone_with_mysqlrouter(
-            ops_test, ANOTHER_KEYSTONE_APP_NAME, ANOTHER_KEYSTONE_MYSQLROUTER_APP_NAME, 2
+        # Deploy keystone
+        # Explicitly setting the series to 'focal' as it defaults to 'xenial'
+        another_keystone_app = await ops_test.model.deploy(
+            "keystone",
+            series="focal",
+            application_name=ANOTHER_KEYSTONE_APP_NAME,
+            num_units=2,
+        )
+
+        # Deploy mysqlrouter and relate it to keystone
+        another_keystone_mysqlrouter_app = await ops_test.model.deploy(
+            "mysql-router",
+            channel="8.0/stable",  # pin to channel as it contains a fix to https://bugs.launchpad.net/charm-mysql-router/+bug/1927981
```

Comment:
I believe this is temporary. Let's raise a placeholder issue to remove this once the fix propagates to latest/default channel.

Reply:
good idea! created a placeholder issue [here](https://github.com/canonical/mysql-operator/issues/45)